modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -83,7 +83,9 @@ If your application does not expose its data on the default public port, 80, cle
 . Optional: You can use the following advanced options to further customize your application:
 
 include::snippets/routing-odc.adoc[]
+ifndef::openshift-rosa,openshift-dedicated[]
 include::snippets/serverless-domain-mapping-odc.adoc[]
```

Comment:
if I'm understanding correctly, you are hiding serverless domain mapping in OSD & ROSA, which is showing up corrrectly. But there is still a leftover xref in the "additional resources" for domain mapping for ROSA & OSD (second bullet in screenshot).
![Screenshot 2024-02-29 at 11 06 26 AM](https://github.com/openshift/openshift-docs/assets/96192119/f857d471-82ee-4c01-b4b7-7d3ac3494611)
But I might not be interpreting this correctly, so please ignore if that is the case.

Reply:
@kcarmichael08 Correct. I am hiding serverless domain mapping in ROSA/OSD, as the user cannot install a required Operator. However, there are other domain mapping methods they can use.  (You almost stumped me on this one!)